### ISSUE TITLE
Files that were not fixed due to internal error (PHP CS Fixer version 1.7)

### ISSUE THREAD

**Issue by eusonlito** [FriendsOfPHP/PHP-CS-Fixer#1164]
Hi, same again using PHP CS Fixer version 1.7

File to fix: https://github.com/eusonlito/braintree_php/blob/master/src/Braintree/Error/ValidationErrorCollection.php

Error:
```
php-cs-fixer -vvv fix src/Braintree/Error/ValidationErrorCollection.php

E
Legend: ?-unknown, I-invalid file syntax, file ignored, .-no changes, F-fixed, E-error
Fixing time per file:
[1.002 s] ./src/Braintree/Error/ValidationErrorCollection.php

Fixed all files in 1.011 seconds, 5.750 MB memory used

Files that were not fixed due to internal error:
   1) ./src/Braintree/Error/ValidationErrorCollection.php
```

Thanks :)

**Comment by keradus** [comment]
Are you using any `.php_cs` configuration?

**Comment by eusonlito** [comment]
No, I haven't any `.php_cs` config file

**Comment by eusonlito** [comment]
My PHP configuration:

```
$ php -v
PHP 5.5.12-2ubuntu4.3 (cli) (built: Mar 16 2015 20:50:26) 
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2014 Zend Technologies
    with Zend OPcache v7.0.4-dev, Copyright (c) 1999-2014, by Zend Technologies
```

**Comment by keradus** [comment]
Fix: #1165

**Comment by keradus** [comment]
Closing due to merge of #1165
